packages/neos-ui/src/Containers/PrimaryToolbar/MenuToggler/style.css

```diff
@@ -42,3 +57,11 @@
         color: var(--colors-PrimaryBlue);
     }
 }
+
+.menuToggler__logo {
+    position: absolute;
+    top: 8px;
```

Comment:
Same here we can use the constants and calculations of it.
e.g. var(--spacing-Half) for 8 pixels.

Would it help when I create a cheat sheet with all constants and values?

Reply:
Thank you, I know there is a file with all the constants 😉 But as e.g. the spacing constants are not semantic in any way, I am careful in using them – because it might break later on if someone decides "half" to be 12px instead of 8px.

Might make sense to introduce semantic constants, like "componentPadding", "toolbarPadding" or similar.

Reply:
@danielkestler that's a very good point!
Though `toolbarPadding` might be a bit too specific, but I like the general idea.

Anyways, I'm not super keen on using spacing constants either, but either way we should try to be consistent: if we don't want them, let's just get rid of them. Or use your suggestion.

In anycase let's not overengineer, do what feels best to you :)

Reply:
I don't know if we have a proper grid system. If so it would be nice to have all values we need and use them everywhere. With a robust system you can change the look and feel of neos backend with just a couple values. But therefore the whole code base have to use the constants and variantions of them. So eliminating static values would be nice from my point of view.

But maybe discuss is a better place to talk about that then a pull request :D

Reply:
No longer applies ... ;-)